src/NewProject/ProjectManager/project_manager.cpp

```diff
@@ -1418,22 +1418,6 @@ int ProjectManager::CreateVerilogFile(QString strFile) {
   if (!file.open(QFile::WriteOnly | QFile::Text)) {
     return -1;
   }
-  QFile templ{":/templates/verilog_templ.v"};
```

Comment:
Are there any templates on the upsteam codebase ?

Reply:
no

Reply:
We also need to make sure no code calls CreateVerilogFile automatically.
Only if the user adds a file this should happen.
It looks like some actions are calling CreateVerilogFile, please do a search in the code.

Reply:
I think we are good.
https://github.com/os-fpga/FOEDAG/search?q=CreateVerilogFile

Reply:
Yes, there are few places that potential could create new file. I'm on it

Reply:
@alaindargelas @alain-rs didn't found any cases. If you provide steps I'll debug that